tools/test/test262/excludelist.orig.xml

```diff
@@ -967,58 +967,8 @@
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/get-length-ignores-length-prop"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-call-changes-value"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-call-parameters"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-call-this-non-strict"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-call-this-strict"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-may-detach-buffer"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/predicate-not-called-on-empty-array"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/return-abrupt-from-predicate-call"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/return-index-predicate-result-is-true"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/BigInt/return-negative-one-if-predicate-returns-false-value"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/get-length-ignores-length-prop"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/invoked-as-func"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/invoked-as-method"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/length"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/name"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-call-changes-value"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-call-parameters"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-call-this-non-strict"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-call-this-strict"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-may-detach-buffer"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/predicate-not-called-on-empty-array"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/prop-desc"><reason>TODO</reason></test>
-    <test id="built-ins/TypedArray/prototype/findLastIndex/return-abrupt-from-predicate-call"><reason>TODO</reason></test>
     <test id="built-ins/TypedArray/prototype/findLastIndex/return-abrupt-from-this-out-of-bounds"><reason>TODO</reason></test>
```

Comment:
Can I ask you something?
Why this test case is remained? I want to know the reason of this

Reply:
This test includes the following code
```js
assert.sameValue(                                                                                                                                                                      
  typeof ArrayBuffer.prototype.resize,
  'function',
  'implements ArrayBuffer.prototype.resize'
);
```

`ArrayBuffer.prototype.resize` is one of proposal features but not yet implemented in Escargot.